sources/install/package_network.sh

```diff
@@ -175,3 +176,21 @@ function install_tailscale() {
     add-test-command "tailscale --help"
     add-to-list "tailscale,https://github.com/tailscale/tailscale,A secure and easy-to-use VPN alternative that is designed for teams and businesses."
 }
+
+function install_nfspy(){
+    colorecho "Installing nfspy"
+    git -C /opt/tools/ clone https://github.com/bonsaiviking/NfSpy.git
+    cd /opt/tools/NfSpy
+    fapt libfuse-dev
+    virtualenv -p /usr/bin/python2 ./venv
+    ./venv/bin/python2 -m pip install fuse-python
+    ./venv/bin/python2 setup.py install
+    chmod +x scripts/*
+    sed -i "s#/usr/bin/env python#/opt/tools/NfSpy/venv/bin/python2#" /opt/tools/NfSpy/scripts/nfspy
+    sed -i "s#/usr/bin/env python#/opt/tools/NfSpy/venv/bin/python2#" /opt/tools/NfSpy/scripts/nfspysh
+    ln -v -s /opt/tools/NfSpy/scripts/nfspy /opt/tools/bin/nfspy
+    ln -v -s /opt/tools/NfSpy/scripts/nfspysh /opt/tools/bin/nfspysh
+    add-test-command "nfspy --help"
+    add-test-command "nfspysh --help"
+    add-history nfspy
+}
```

Comment:
It missing the `add-to-list` command to reference the tool to the documentation.

For example :
<img width="904" alt="image" src="https://user-images.githubusercontent.com/51704860/235158997-ff40f18a-ce2b-4d6b-83a0-319a0bf2e4f8.png">